src/main/resources/application.yaml

```diff
@@ -25,6 +25,8 @@ spring:
           order_inserts: true
   application:
     name: RD Caseworker Ref API
+  main:
+    allow-bean-definition-overriding: true
   security:
```

Comment:
Can below 
`- serviceauthorization` be removed from 
`feign:
  allow:
    headers:
      - authorization
      - serviceauthorization`
- https://github.com/hmcts/rd-caseworker-ref-api/blob/652cc9ca2f26767154bb53b82d09e38c8e1582ff/src/main/resources/application.yaml#L106.

as we don't pass the service header from CRD to other service now ?

Reply:
Good catch Sujeeth.
Removed ServiceAuth from application.yaml.